syntax = "proto3";

message AppendEntries
{
    int32 term = 1;
    int32 leaderId = 2;
    int32 prevLogIndex = 3;
    int32 prevLogTerm = 4;
    repeated string entries = 5;
    int32 leaderCommit = 6;
}